shell/browser/ui/cocoa/atom_bundle_mover.h

```diff
@@ -11,6 +11,9 @@
 
 namespace electron {
 
+// Possible bundle movement conflicts
+enum class ConflictType { EXISTS, EXISTS_AND_RUNNING };
```

Comment:
Just to make this slight more scoped than "an electron conflict"

```suggestion
enum class BundlerMoverConflictType { EXISTS, EXISTS_AND_RUNNING };
```